src/index.php

```diff
@@ -0,0 +1,18 @@
+<?php
+require 'functions.php';
+
+echo produce_HTML_Head();
```

Comment:
this shouldn't really be a function because it causes tight coupling by hiding away the html in the php, better to put it as plain html at the bottom of this file after the php section

Reply:
HTML structure now sits inline, rather than generated by PHP